playbooks/group_vars/all/global.yml

```diff
@@ -3,6 +3,7 @@
 run_cleanup: false
 run_prereqs: true
 setup_minishift: true
+startup_minishift: false
```

Comment:
This should be changed to `start_minishift` -- or alternatively the other references to `start_minishift` should be changed to `startup_minishift`.

Reply:
@psiroky good catch will have a fix in a min